source/SIL.AppBuilder.Portal.Frontend/src/ui/components/errors/index.ts

```diff
@@ -1,3 +1,4 @@
 export { default as PageError } from './page';
 export { parseError } from './parse-error';
 export { default as ErrorMessage } from './header-message';
```

Comment:
why not do export ErrorMessage from './header-message' ?

Reply:
because it doesn't have a named export, and the default export, by convention, is the default thing the file defines